tests/filtertransformers/test_elasticsearch.py

```diff
@@ -69,16 +69,16 @@ def test_parse_n_transform(self):
             ('chemical_formula_reduced STARTS WITH "H2"', 3),
             ('chemical_formula_reduced ENDS WITH "C"', 1),
             ('chemical_formula_reduced ENDS "C"', 1),
-            ("LENGTH elements = 2", 3),
-            ("LENGTH elements = 3", 1),
-            ("LENGTH dimension_types = 0", 3),
-            ("LENGTH dimension_types = 1", 1),
-            ("nelements = 2 AND LENGTH dimension_types = 1", 1),
-            ("nelements = 3 AND LENGTH dimension_types = 1", 0),
-            ("nelements = 3 OR LENGTH dimension_types = 1", 2),
-            ("nelements > 1 OR LENGTH dimension_types = 1 AND nelements = 2", 4),
-            ("(nelements > 1 OR LENGTH dimension_types = 1) AND nelements = 2", 3),
-            ("NOT LENGTH dimension_types = 1", 3),
+            ("elements LENGTH = 2", 3),
+            ("elements LENGTH = 3", 1),
+            ("dimension_types LENGTH = 0", 3),
+            ("dimension_typesLENGTH = 1", 1),
+            ("nelements = 2 AND dimension_types LENGTH = 1", 1),
+            ("nelements = 3 AND dimension_types LENGTH = 1", 0),
+            ("nelements = 3 OR dimension_types LENGTH = 1", 2),
+            ("nelements > 1 OR dimension_typesLENGTH = 1 AND nelements = 2", 4),
```

Comment:
```suggestion
            ("nelements > 1 OR dimension_types LENGTH = 1 AND nelements = 2", 4),
```

Reply:
Thats weird. It should have passed the tests, right?

Reply:
The grammar (I guess this is how its specified) only uses lower case letters for properties and the parser therefore can separate tokens `dimension_types` and `LENTGH` even without a whitespace between.